docs/docs/cmd/docs.md

```diff
@@ -0,0 +1,31 @@
+# docs
+
+Opens CLI for Microsoft 365 docs webpage in the default browser.
+
+## Usage
+
+```sh
+m365 docs [options]
+```
+
+## Options
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Opens CLI for Microsoft 365 docs webpage in the default browser.
+
+```sh
+m365 docs
+```
+
+Opens CLI for Microsoft 365 docs webpage in the default browser and logs detailed debug information in the console output
+
+```sh
+m365 docs --debug
```

Comment:
This example doesn't add much value, let's remove this one. If there are no more useful examples, 1 will suffice.

Reply:
Removed the unnecessary example.